### DISCUSSION TITLE
How to remove code from a old freelancer project that was added to Github

### DISCUSSION THREAD

**Discussion by simpleraison** [community/community#45715] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi,

I completed a project on a freelancer website some times ago, the worker used the Github website to add files of our code to work on. This was a long time ago and since then we have lost contact with the worker, the code still remains on the Github website, how are we able to get this removed?

Thank you.

**Comment by jge162** [comment]
If you do not own the repository, you will need to get in contact with the owner, create a fork then a pull request in an effort to reach him.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @simpleraison ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
